torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -912,7 +921,7 @@ RegisterOperators reg(
            // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
            int64_t a, b;
            pop(stack, a, b);
-           push(stack, pow(a, b));
+           push(stack, powWrapper(a, b));
```

Comment:
For the case of `int a ** int b` in Python, `DEFINE_GENERIC_OP_WITH_COMPLEX` is used to resolve the aten::pow. I am not sure how to test for this operator `aten::pow.int_to_int(int a, int b) -> int` as I am not sure how to trigger this in Python

Reply:
Yeah, that seems to be the case. Since our overall goal is to align the two different behaviors, I'd suggest to just replace all pow call to the wrapper call, and then I'll figure out whether they're used later and do something in a separate diff.